src/pages/workspace/categories/CategorySettingsPage.tsx

```diff
@@ -0,0 +1,87 @@
+import NotFoundPage from '@pages/ErrorPage/NotFoundPage';
+import AdminPolicyAccessOrNotFoundWrapper from '@pages/workspace/AdminPolicyAccessOrNotFoundWrapper';
+import PaidPolicyAccessOrNotFoundWrapper from '@pages/workspace/PaidPolicyAccessOrNotFoundWrapper';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type SCREENS from '@src/SCREENS';
+import type * as OnyxTypes from '@src/types/onyx';
+
+type CategorySettingsPageOnyxProps = {
+    /** Collection of categories attached to a policy */
+    policyCategories: OnyxEntry<OnyxTypes.PolicyCategories>;
+};
+
+type CategorySettingsPageProps = CategorySettingsPageOnyxProps & StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.CATEGORY_SETTINGS>;
+
+function CategorySettingsPage({route, policyCategories}: CategorySettingsPageProps) {
+    const {isSmallScreenWidth} = useWindowDimensions();
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const policyCategory = policyCategories?.[route.params.categoryName];
+
+    if (!policyCategory) {
+        return <NotFoundPage />;
+    }
+
+    const updateWorkspaceRequiresCategory = (value: boolean) => {
+        setWorkspaceCategoryEnabled(route.params.policyID, {[policyCategory.name]: {name: policyCategory.name, enabled: value}});
+    };
+
+    return (
+        <AdminPolicyAccessOrNotFoundWrapper policyID={route.params.policyID}>
+            <PaidPolicyAccessOrNotFoundWrapper policyID={route.params.policyID}>
+                <ScreenWrapper
+                    includeSafeAreaPaddingBottom={false}
+                    style={[styles.defaultModalContainer]}
+                    testID={CategorySettingsPage.displayName}
+                    shouldShowOfflineIndicatorInWideScreen
+                >
+                    <HeaderWithBackButton
+                        title={route.params.categoryName}
+                        shouldShowBackButton={isSmallScreenWidth}
+                    />
+
+                    <View style={[styles.mt2, styles.mh5]}>
+                        <View style={[styles.flexRow, styles.mb5, styles.mr2, styles.alignItemsCenter, styles.justifyContentBetween]}>
+                            <Text>{translate('workspace.categories.enableCategory')}</Text>
+                            <Switch
+                                isOn={policyCategory.enabled}
+                                accessibilityLabel={translate('workspace.categories.enableCategory')}
+                                onToggle={updateWorkspaceRequiresCategory}
+                            />
+                        </View>
+                    </View>
+                    <MenuItemWithTopDescription
+                        title={policyCategory.name}
+                        description={translate(`workspace.categories.categoryName`)}
+                        // TODO: do we want disable menu item if category is disabled?
+                        disabled={!policyCategory.enabled}
```

Comment:
@luacmartins This part was missing in docs, do we want gray out and disable category menu item if it is disabled?

Reply:
No, I don't think so. You should still be able to rename it even if it's disabled.